### PR TITLE
[tests-only] added spaces examples in the api WebDAV locks suite

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
Added spaces examples in the suite `apiWebdavLocks`

## Related Issue
- Part of https://github.com/owncloud/ocis/issues/3098

## How Has This Been Tested?

- :robot: 
- https://github.com/owncloud/ocis/pull/3108

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests only (no source changes)

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [ ] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiAuth-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/47/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiAuthWebDav-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/49/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiMain-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/53/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiShareCreateSpecialToRoot1-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/58/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiFavorites-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/52/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiCapabilities-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/50/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiWebdavLocks2-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/89/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiWebdavLocks3-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/90/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiWebdavPreviews-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/95/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>cliExternalStorage-git-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/122/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiFederationToShares2-latest-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/112/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiFederationToShares1-git-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/108/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiFederationToRoot2-10.8.0-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/107/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>cliMain-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/116/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIAddUsers-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34510/125/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIUpload-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34511/151/1

File: tests/acceptance/features/apiWebdavLocks/exclusiveLocks.feature [hunk 1]
```diff
@@ -76,16 +96,23 @@ Feature: there can be only one exclusive lock on a resource
       | new      | shared     |
       | new      | exclusive  |
 
+    @personalSpace @skipOnOcV10
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiWebdavLocks/exclusiveLocks.feature:99]:
```suggestion
    @personalSpace 
```

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIUpload-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34517/151/1

**Comment by kiranparajuli589** [comment]
```console
@webUI @insulated @disablePreviews @files_sharing-app-required
Feature: File Upload
  
  As a user
  I would like to be able to upload files via the WebUI
  So that I can store files in ownCloud

  Background:                                                                              # /drone/src/tests/acceptance/features/webUIUpload/upload.feature:8
    Given user "Alice" has been created with default attributes and without skeleton files # FeatureContext::userHasBeenCreatedWithDefaultAttributesAndWithoutSkeletonFiles()

  @files_sharing-app-required
  Scenario: upload overwriting a file into a public share                                               # /drone/src/tests/acceptance/features/webUIUpload/upload.feature:131
    Given user "Alice" has created folder "/simple-folder"                                              # FeatureContext::userHasCreatedFolder()
    And user "Alice" has uploaded file "filesForUpload/lorem.txt" to "/simple-folder/lorem.txt"         # FeatureContext::userHasUploadedAFileTo()
    And user "Alice" has created a public link share with settings                                      # FeatureContext::userHasCreatedAPublicLinkShareWithSettings()
      | path        | /simple-folder     |
      | permissions | read,update,create |
    When the public accesses the last created public link using the webUI                               # WebUISharingContext::thePublicAccessesTheLastCreatedPublicLinkUsingTheWebUI()
INFORMATION: timed out waiting for ajax calls to start
Upload overwriting lorem.txt and got 1 notifications including You don’t have permission to upload or create files here

INFORMATION: timed out waiting for ajax calls to start
    And the user uploads overwriting file "lorem.txt" using the webUI and retries if the file is locked # WebUIFilesContext::theUserUploadsOverwritingFileUsingTheWebUIRetry()
      │ INFORMATION: timed out waiting for ajax calls to startUpload overwriting lorem.txt and got 1 notifications including You don’t have permission to upload or create files here
      │ INFORMATION: timed out waiting for ajax calls to start
      WebUIFilesContext::choiceInUploadConflictDialogWebUI file upload conflict dialog expected but not found (Exception)
    Then file "lorem.txt" should be listed on the webUI                                                 # WebUIFilesContext::fileFolderShouldBeListedOnTheWebUI()
    When user "Alice" logs in using the webUI                                                           # WebUILoginContext::logInWithUsernameUsingTheWebUI()
    And the content of "simple-folder/lorem.txt" should be the same as the local "lorem.txt"            # WebUIFilesContext::theContentOfShouldBeTheSameAsTheLocal()
SCENARIO RESULT: (fail)

```

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>apiWebdavLocks-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34521/88/1

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIUpload-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34523/151/1

File: tests/acceptance/features/apiWebdavLocks/requestsWithToken.feature [hunk 2]
```diff
@@ -112,20 +137,26 @@ Feature: actions on a locked item are possible if the token is sent with the req
...
+    Then the HTTP status code should be "204"
     And the content of file "textfile0.txt" for user "Alice" should be "from user 1"
-    And the content of file "textfile0 (2).txt" for user "Brian" should be "from user 1"
+    And the content of file "Shares/textfile0.txt" for user "Brian" should be "from user 1"
     Examples:
       | dav-path |
       | old      |
       | new      |
+
+    @personalSpace  @skipOnOcV10
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiWebdavLocks/requestsWithToken.feature:159]:
```suggestion
    @personalSpace 
```

File: tests/acceptance/features/apiWebdavLocks/requestsWithToken.feature [hunk 3]
```diff
@@ -112,20 +137,26 @@ Feature: actions on a locked item are possible if the token is sent with the req
     And user "Brian" has been created with default attributes and without skeleton files
     And user "Alice" has uploaded file with content "some data" to "textfile0.txt"
     And user "Brian" has uploaded file with content "some data" to "textfile0.txt"
-    And user "Alice" has shared file "textfile0.txt" with user "Brian"
+    And user "Alice" has shared file "/textfile0.txt" with user "Brian"
+    And user "Brian" has accepted share "/textfile0.txt" offered by user "Alice"
     And user "Alice" has locked file "textfile0.txt" setting the following properties
       | lockscope | shared |
-    And user "Brian" has locked file "textfile0 (2).txt" setting the following properties
+    And user "Brian" has locked file "Shares/textfile0.txt" setting the following properties
```
Comment (amrita-shrestha) [inline tests/acceptance/features/apiWebdavLocks/requestsWithToken.feature:144]: **`And the administrator has set the default folder for received shares to "Shares"`**
I think this step is needed to stored share file  inside `/Shares` folder in core

Comment (kiranparajuli589) [reply tests/acceptance/features/apiWebdavLocks/requestsWithToken.feature:144]: this will never run on core so does not make any difference

Comment (amrita-shrestha) [reply tests/acceptance/features/apiWebdavLocks/requestsWithToken.feature:144]: nice

**Review by amrita-shrestha — COMMENTED** [review]
LGTM

**Comment by ownclouders** [comment]
:boom: Acceptance tests pipeline <strong>webUIUpload-chrome-mariadb10.2-php7.4</strong> failed. The build has been cancelled.

https://drone.owncloud.com/owncloud/core/34534/151/1

**Review by phil-davis — APPROVED** [review]

**Comment by phil-davis** [comment]
I rebased to get up-to-date CI. This can be merged when CI is green

**Pull request merged**